src/xpcc/architecture/platform/driver/core/cortex/syscalls.cpp.in

```diff
@@ -290,11 +291,13 @@ void *malloc(size_t size)
 	return p;
 }
 
+__attribute__((used))
 void *realloc(void *p, size_t size)
 {
 	return tlsf_realloc(get_tlsf_for_ptr(p), p, size);
 }
 
+__attribute__((used))
```

Comment:
Yeah, this is the brute-force method to get something working.
And indeed, blinky links in the TLSF code, even though it does not use it.
Hmpf.